tests/integration/io/parquet/test_reads_public_data.py

```diff
@@ -19,6 +19,8 @@ def get_filesystem_from_path(path: str, **kwargs) -> fsspec.AbstractFileSystem:
     return fs
 
 
+# TODO(desmond): Expected some of these non-working cases to work after recent fixes, but that
+#                doesn't seem to be the case.
```

Comment:
We should be able to enable some of the struct tests after fixing the casting issues that we saw yesterday.